itou/metabase/management/commands/_database_tables.py

```diff
@@ -20,13 +20,6 @@ def get_old_table_name(table_name):
     return f"z_old_{table_name}"
 
 
-def get_dry_table_name(table_name):
```

Comment:
Typo dans le commit 7deb2fb4232a06d3844f5061395da3918d7715a7 : `confusoing`, et `Open to comments !/Open to comments!`